db/migrate/20180214163355_populate_worflow_on_existing_cases.rb

```diff
@@ -0,0 +1,15 @@
+class PopulateWorflowOnExistingCases < ActiveRecord::Migration[5.0]
+  def up
+    Case::Base.find_each do |kase|
+      if kase.type_abbreviation == 'FOI' && kase.flagged?
+        kase.update(workflow: 'trigger')
+      else
+        kase.update(workflow: 'standard')
```

Comment:
I'm asking myself if `standard` the best wording here. FOI cases have the sub-types `Standard`, `TimelinessReview` and `ComplianceReview`. And then there's SAR cases ... maybe we should use `Default` to be clear we're not talking about `Case::FOI::Standard`.

Reply:
I'm easy, but it's in the workflow column on case, and to_workflow column on case_transition, so don't think there is much room for confusion.